docs/platform/concepts/orgs-units-projects.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Organizations, units, and projects
+---
+
+The Aiven platform uses organizations, organizational units, and projects to organize services and manage access.
+
+import Overview from "@site/static/images/content/figma/organizations-overview.png";
+
+<img alt="Hierarchy showing two organizational units, each with two projects, nested within one organization." src={Overview} width="58%" class="centered"/>
+
+## Organizations and organizational units
```

Comment:
I would suggest to separate these as they don't come together but one is an extension of another. We can then clearly outline what an Organization is and what an unit is.